### PR TITLE
chore(docs): Getting started, revisited + cookie banner - FRONT-4053

### PR DESCRIPTION
**Pull request by planctus**

### CODE REVIEW

File: src/website/src/pages/ec/getting-started/index.mdx [hunk 1]
```diff
@@ -19,7 +19,52 @@ New components are continuously being added to the library. The team is also con
...
+`npm install @ecl/preset-ec`
+OR
+`yarn add @ecl/preset-ec`
+
+### Alternative approaches:
+
+ECL is also distributing its presets on a CDN, all the files distributed in the package mentioned before are also available in the CDN, this is the pattern to reach them:
+`https://cdn{1,2,3 or 4}.fpfis.tech.ec.europa.eu/ecl/{tag}/{system}/{path/to/the/asset}`
+
+Ex: https://cdn1.fpfis.tech.ec.europa.eu/ecl/v3.9.1/ec/styles/optional/ecl-ec-default.css
```
Comment (emeryro) [inline src/website/src/pages/ec/getting-started/index.mdx]: Is there a way to somehow expose the file structure on the cdn (automatically or not)?
Without knowing the exact path it is quite hard to guess it if you don't know the folder/file names
That would be useful to just list the different css/js files (if we don't have this doc already somewhere...)

File: src/website/src/pages/ec/getting-started/index.mdx [hunk 2]
```diff
@@ -19,7 +19,52 @@ New components are continuously being added to the library. The team is also con
...
+
+These resources can be used loading them directly from the CDN, with the limitation that it's not possible to serve svgs from an external domain without using some workaround, see the "Additional information" below.
+
+For different reasons, starting from performance and network traffic, **the recommended way of using ECL is to fetch its resources for then hosting them "locally"**.
```
Comment (emeryro) [inline src/website/src/pages/ec/getting-started/index.mdx]: I'd suggest splitting the "How to use ECL" section into somethig like "Using npm packages", "Using offline/local files" and "Using online files/cdn"

File: src/website/src/pages/ec/getting-started/index.mdx [hunk 3]
```diff
@@ -19,7 +19,52 @@ New components are continuously being added to the library. The team is also con
...
+
+In order to ease the access to the package containing them, the main package mentioned before: `preset-ec` is also deployed in the github release page and can be downloaded from there.
+
+Here is an example of a default HTML file using ECL:
```
Comment (emeryro) [inline src/website/src/pages/ec/getting-started/index.mdx]: We could put that in a dedicated section, this is not really related to one way to use ECL or the other

File: src/website/src/pages/ec/getting-started/index.mdx [hunk 4]
```diff
@@ -36,34 +81,28 @@ Here is an example of default HTML file using ECL:
...
   </head>
   <body>
     <!-- content of your page here -->
 
-    <script
-      src="https://cdn1.fpfis.tech.ec.europa.eu/ecl/[ECL_VERSION]/ec/scripts/ecl-ec.js"
-      integrity="[sri hash of ecl-ec.js]"
-      crossorigin="anonymous"
-    ></script>
+    <script src="/scripts/ecl-ec.js"></script>
```
Comment (emeryro) [inline src/website/src/pages/ec/getting-started/index.mdx]: Shouldn't we also add some dummy path to moment.js in this example?

File: src/website/src/pages/ec/getting-started/index.mdx [hunk 5]
```diff
@@ -73,7 +112,8 @@ Therefore **moment.js needs to be loaded or bundled by the application or websit
 
 - If you wish to use ECL EU instead of ECL EC, you can find information here <Link to="/eu/getting-started/" label="Getting started EU" />
 - Replace <code>[ECL_VERSION]</code> by <code>v{process.env.ECL_VERSION}</code> to use the latest version
```
Comment (emeryro) [inline src/website/src/pages/ec/getting-started/index.mdx]: This seems outdated (no more [ECL_VERSION])

**Review by emeryro — DISMISSED** [review]
I added a few remarks/suggestions, I leave it up to you

File: src/website/src/pages/ec/getting-started/index.mdx (hunk 2 above)
Comment (planctus) [reply src/website/src/pages/ec/getting-started/index.mdx]: mmh.. not sure, i would find it very confusing, in the end there is just ONE way of using ECL, what changes is how the resources are fetched or simply linked.
So, at the moment the different alternatives are under the "Where to get the ECL resources" heading, but i agree that the section can be improved. Will try to do it

File: src/website/src/pages/ec/getting-started/index.mdx (hunk 3 above)
Comment (planctus) [reply src/website/src/pages/ec/getting-started/index.mdx]: no indeed, that is the reason why the example is at the end, because that is the (unique) way to use ECL once you have decided whether to fetch the resources or use the CDN.

File: src/website/src/pages/ec/getting-started/index.mdx (hunk 4 above)
Comment (planctus) [reply src/website/src/pages/ec/getting-started/index.mdx]: yep, makes sense

File: src/website/src/pages/ec/getting-started/index.mdx [hunk 6]
```diff
@@ -34,92 +52,76 @@ Here is an example of default HTML file using ECL:
...
 </details>
 
+When using the CDN for icons and logos be aware that it's not possible to serve svgs from an external domain without using some workaround, see the "Additional information" below.
+
+For different reasons, starting from performances and network traffic, **the recommended way of using ECL is to fetch its resources for then hosting them in the same domain where they are going to be used**.
+
 ### Optional styles
 
-There are several optional CSS files, located in `ec/styles/optional`, which can be included in your site on not, depending on the needs.  
+There are several optional CSS files, located in `eu/styles/optional`, which can be included in your site on not, depending on the needs.
```
Comment (emeryro) [inline src/website/src/pages/ec/getting-started/index.mdx]:
```suggestion
There are several optional CSS files, located in `ec/styles/optional`, which can be included in your site on not, depending on the needs.  
```

**Review by emeryro — DISMISSED** [review]

**Review by emeryro — APPROVED** [review]
